parlai/agents/seq2seq/seq2seq.py

```diff
@@ -150,6 +152,7 @@ def __init__(self, opt, shared=None):
         self.truncate = opt['truncate'] if opt['truncate'] > 0 else None
         self.metrics = {'loss': 0.0, 'num_tokens': 0}
         self.history = {}
+        self.report_freq = opt['report_freq']
```

Comment:
can you make this opt.get('report_freq', 0.001) just to make sure old loaded opts don't freak out?

Reply:
oh I'll comment on the other one

Reply:
right, thanks!